src/pages/styles/colors.tsx

```diff
@@ -14,17 +14,20 @@ import Section from 'components/Section'
 import ComponentExample from 'components/ComponentExample'
 import { Heading, Text, Paragraph } from 'components/ResponsiveComponents'
 
-const borderForLightColor = `1px solid ${suomifiTheme.colors.depthLight13}`
+const colorTokens =
+  !!suomifiTheme && !!suomifiTheme.colors
+    ? suomifiTheme.colors
+    : { depthLight13: undefined }
```

Comment:
What is the side effect of this condition returning the defined "mock" object? Would failing build be better?

Reply:
This is actually a fallback for Typescript as cannot create `borderForLightColor` (https://github.com/vrk-kpa/suomifi-design-system/pull/69/files#diff-15b9edbc32a3cba6adfa1f3fd830e7caR22) without fallback. The fallback for that could be done when it is used but I'm already checking the condition here, would have done with function attribute defaults if this would have been done in a functional way

Reply:
I'm open to that this can be refactored later on. Trying to not make this PR any larger.